dotnet/src/SemanticKernel.Skills/Skills.Memory.VectorDB/Qdrant/DotNet/Internal/Diagnostics/Verify.cs

```diff
@@ -0,0 +1,207 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
This is tough. For context, we had originally hoped that the vector DB connectors would be deployed via their own packages. We've been hesitant to tie these connectors too closely with the Kernel. I think Shawn should weigh in here. 

Agree that it would be much cleaner to use the Kernel utilities, unclear if it makes long-term sense though.

Reply:
> be deployed via their own packages

Meaning this project may someday not reference Microsoft.SemanticKernel at all? I'd like to learn more if so.